File: MTWDM iOS Xamarin/App2/App1/ViewController.cs
using System;

using UIKit;

namespace App1
{
    public partial class ViewController : UIViewController
    {

        int contador = 0;

        protected ViewController(IntPtr handle) : base(handle)
        {
            // Note: this .ctor should not contain any initialization logic.

        }

        public override void ViewDidLoad()
        {
            base.ViewDidLoad();
            // Perform any additional setup after loading the view, typically from a nib.

            inicializar();
            eventos();
        }

        public void inicializar(){

        }

        public void eventos(){

            btnonClic.TouchUpInside += delegate {

                contador = contador + 1;

                lblNumero.Text = contador.ToString();

            };

            btnonresetear.Clicked += Btnonresetear_Clicked;

        }

        void Btnonresetear_Clicked(object sender, EventArgs e)
        {
            contador = 0;
            lblNumero.Text = "0";

        }

        partial void onContarPress(UIKit.UILongPressGestureRecognizer sender){
            contador = contador + 1;

            lblNumero.Text = contador.ToString();
        }


        public override void DidReceiveMemoryWarning()
        {
            base.DidReceiveMemoryWarning();
            // Release any cached data, images, etc that aren't in use.
        }
    }
}

File: MTWDM iOS Swift /app2/app1/ViewController.swift
//
//  ViewController.swift
//  app1
//
//  Created by Felipe Hernandez on 07/09/18.
//  Copyright © 2018 Felipe Hernandez. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    var contador = 0
    
    @IBAction func onContadorPress(_ sender: UILongPressGestureRecognizer) {
        
        contador =  contador + 1
        lblContador.text  =  String(contador)
        
    }
    
    @IBAction func onResetear(_ sender: Any) {
        contador = 0
        lblContador.text  = "0"
    }
    @IBOutlet weak var lblContador: UILabel!
    
    @IBAction func onContador(_ sender: Any) {
        contador =  contador + 1
        lblContador.text  =  String(contador)
    }
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        lblContador.text = String(contador)
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

